arrow/arrow.py

```diff
@@ -141,7 +141,7 @@ def now(cls, tzinfo=None):
         )
 
     @classmethod
-    def utcnow(cls):
+    def utcnow(cls) -> "Arrow":
```

Comment:
Arrow needs to be quoted because you are returning the type defined in this file right?